src/numberGenerator.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function numberGenerator() {
+  const arrayOfRandomNumbers = [];
+
+  while (arrayOfRandomNumbers.length < 4) {
+    const random = Math.floor(0 + Math.random() * (9 + 1 - 0)).toString();
+
+    if (arrayOfRandomNumbers.includes(random)) {
+      continue;
+    }
+
+    arrayOfRandomNumbers.push(random);
+  }
+
+  return arrayOfRandomNumbers.join('');
```

Comment:
Why do you use a `join` here and convert the array to a string, so that later in the `app.js` file you call `split` and return it back to the array?

Reply:
> since I need both a string and an array

It's strange behavior in this case, but ok